Sources/XMLCoder/Auxiliaries/XMLElement.swift

```diff
@@ -209,3 +209,21 @@ internal class _XMLElement {
         return string
     }
 }
+
+extension _XMLElement: Equatable {
+    static func == (lhs: _XMLElement, rhs: _XMLElement) -> Bool {
+        guard lhs.key == rhs.key else {
+            return false
+        }
+        guard lhs.value == rhs.value else {
+            return false
+        }
+        guard lhs.attributes == rhs.attributes else {
+            return false
+        }
+        guard lhs.children == rhs.children else {
```

Comment:
would it be possible to merge all of the equality comparisons into a single `guard` block given that all of them do the same `return false`?

Reply:
Sure. Fixed.